corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/utils_spec.js

```diff
@@ -35,7 +35,7 @@ hqDefine("cloudcare/js/form_entry/spec/utils_spec", function () {
                 textInGroup = fixtures.textJSON({ix: "1,0"}),
                 group = fixtures.groupJSON({ix: "1", children: [textInGroup]}),
                 textInRepeat = fixtures.textJSON({ix: "2_0,0"}),
-                groupInRepeat = fixtures.groupJSON({ix: "2_0", children: [textInRepeat]}),
+                groupInRepeat = fixtures.groupJSON({ix: "2_0", children: [textInRepeat], caption: null}),
```

Comment:
adding caption value here seems to circumvent the point of the test. That should be removed, and the change i suggested in the [other comment ](https://github.com/dimagi/commcare-hq/blob/96d92204f04a265cecabbc34880c33e70ac79a90/corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js#L581C17-L581C17) should resolve this.

Reply:
https://github.com/dimagi/commcare-hq/pull/33834/commits/dbab40c2c0c15313bc355da9910ac8ee96dcb24d